services/horizon/internal/ingest/processors/effects_processor.go

```diff
@@ -986,6 +988,22 @@ func (e *effectsWrapper) addClawbackClaimableBalanceEffects() error {
 		history.EffectClaimableBalanceClawedBack,
 		details,
 	)
+
+	// Generate the account credited effect (although the funds will be burned) for the asset issuer
+	for _, c := range changes {
+		if c.Type == xdr.LedgerEntryTypeClaimableBalance && c.Post == nil && c.Pre != nil {
+			cb := c.Pre.Data.ClaimableBalance
+			addAssetDetails(details, cb.Asset, "")
+			details := map[string]interface{}{"amount": amount.String(cb.Amount)}
```

Comment:
Shouldn't these two lines be reversed? Can you add the check to the test?
```suggestion
			details := map[string]interface{}{"amount": amount.String(cb.Amount)}
			addAssetDetails(details, cb.Asset, "")
```

Reply:
Really good catch. I would had hoped this to get caught by the static checker